R/impact_central.R

```diff
@@ -480,13 +468,13 @@ impact_by_year_of_vaccination_activity_type <- function(
                                           c("fvps", "impact"))]
 
   ## Calculate impact
-  fvps_country <- stats::aggregate(fvps ~ country + activity_type + year,
+  fvps_country <- stats::aggregate(fvps ~ country + vaccine + activity_type + year,
```

Comment:
If `vaccine` is required let's add a validation check to call to assert_has_columns. Will external users be coming with data for different vaccines? Should this disaggregation be used in other impact methods?